### DISCUSSION TITLE
Is it possible to prevent the selection tool from resizing?

### DISCUSSION THREAD

**Discussion by TheYoungBeast** [apexcharts/apexcharts.js#4559] [General]
Hi,
Can the selection tool be prevented from resizing so that it can act like a fixed-sized window?
I don't see any appropriate events for that task like a ``beforeSelection``.

![obraz](https://github.com/apexcharts/apexcharts.js/assets/19922252/6bb51b98-31d4-48bf-a09d-6321fc768ece)

I want the user to be able to only move selection from left to right, right to left without being able to resize the selection window.

**Comment by kryst4line** [comment]
I'm trying to do the same, no luck at the moment though. Did you find any way to implement this?

**Comment by psarin** [comment]
Same here, anyone been able to figure it out or a workaround?
